roles/kubernetes-apps/persistent_volumes/openstack/templates/openstack-storage-class.yml

```diff
@@ -8,3 +8,6 @@ metadata:
 provisioner: kubernetes.io/cinder
 parameters:
   availability: nova
+{% if persistent_volumes_type is defined %}
+  type: "{{ persistent_volumes_type }}"
+{% endif %}
```

Comment:
It should be define a value for persistent_volumes_type. and explain its usefulness
Can be defined here( roles/kubernetes-apps/persistent_volumes/openstack/defaults/main.yml)

Reply:
I don't know if we can put a sensible default for this parameter - if it is not defined nothing should happen. We do not know a-priori what cinder device types are or may be defined.
And the name is quite suggestive, I think.

Reply:
my point is to add some explanation.  example
#It needs to select the corresponding cinder device type according to the actual situation of the openstack platform
#ersistent_volumes_type：

Reply:
OK, I added the explanation. Thank you for your help.